### PR TITLE
Create IncidnetReport

### PR DESCRIPTION
**Pull request by Keshav1308**
## Before submitting this PR please ensure that you have read the following sections and then completed the template below:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures, there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

## After the submission has been made, please look at the Validation Checks:

> Check that the validations are passing and address any issues that are present. Let us know if you have tried fixing and need help.

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

## PR Template

-----------------------------------------------------------------------------------------------------------
 **Description for the PR:**
  (Enter the description below)

**Testing Completed:**
  Yes/ No : 

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: Hunting Queries/SecurityIncident/IncidnetReport [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Name: Incident Reports. 
+Author : Keshav Deo Jain
```
Comment (aprakash13) [inline Hunting Queries/SecurityIncident/IncidnetReport:2]: Please remove this field.

**Review by aprakash13 — CHANGES_REQUESTED** [review]
Please model this on the basis of : https://github.com/Azure/Azure-Sentinel/blob/master/Hunting%20Queries/SecurityEvent/ProcessEntropy.yaml

Please correct the typo in the file name.  Probably you wanted the name to be IncidentReport

**Review by Keshav1308 — COMMENTED** [review]
I have made few changes. But Please help. If you can correct it. 

I am not that expert in this github thing. Very often work on this.

**Comment by aprakash13** [comment]
> I have made few changes. But Please help. If you can correct it.
> 
> I am not that expert in this github thing. Very often work on this.

Thanks Keshav1308. I ran the query and feel like that it could use some improvement. For example, if the case is closed same day , it lists in all the three heads that the case was closed on sameday/week/month. Though technically accurate a distinction between whether the case was closed on same day or took more than a day but within a week or more than a week but within a month would perhaps be more helpful in my opinion.

Having said that, you have contributed this to the hunting query. Normally hunting queries are reserved for queries that proactively look for security threats in an environment and are less focused on SOC efficiency numbers or metrics . We do have some SOC efficiency workbook that carry similar information and it might be a great addition to it. Azure-Sentinel/SecurityOperationsEfficiency.json at master · Azure/Azure-Sentinel (github.com). I am closing this for now but hopefully, you will be able to add this to the SecurityOperationsEfficiency workbook. Thanks again Keshav.

**Pull request closed without merging**